.eslintignore

```diff
@@ -0,0 +1 @@
+src/index.d.ts
```

Comment:
no need for this as we'll use `node_modules/eslint/bin/eslint.js src/**/*.js test/**/*.js` in package.json

Reply:
deleted file